.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "thirdparty/PUI"]
```

Comment:
It's good to put it under `thirdparty/` which is an existing directory.  I suggest to also add an automation script under it to document how to pull PUI.